services/simplified_crontab

```diff
@@ -21,9 +21,26 @@ HOME=/var/www/circulation
 # the collections.
 #
 */20 * * * * root core/bin/run bibliographic_coverage >> /var/log/cron.log 2>&1
-*/10 * * * * root core/bin/run metadata_wrangler_coverage >> /var/log/cron.log 2>&1
 30 3 * * * root core/bin/run update_nyt_bestseller_lists >> /var/log/cron.log 2>&1
 
+# Add newly discovered identifiers to our Metadata Wrangler collection.
+*/10 * * * * root core/bin/run metadata_wrangler_collection_sync >> /var/log/cron.log 2>&1
+
+# Remove newly removed identifiers from our Metadata Wrangler collection
+* */22 * * * root core/bin/run metadata_wrangler_collection_reaper >> /var/log/cron.log 2>&1
+
+# Check whether the Metadata Wrangler has ascertained any new
+# information about books in our collection.
+*/59 * * * * root core/bin/run metadata_wrangler_collection_updates >> /var/log/cron.log 2>&1
+
+# If the Metadata Wrangler needs content such as book covers that only we
+# can provide, send it over.
+30 21 * * * root core/bin/run metadata_upload_coverage >> /var/log/cron.log 2>&1
+
+# DEPRECATED: Ask the Metadata Wrangler about all unresolved
+# identifiers, 25 at a time.
+# */10 * * * * root core/bin/run metadata_wrangler_coverage >> /var/log/cron.log 2>&1
```

Comment:
What do you think of making this less frequent, since we're planning to deprecate it?

Reply:
I don't see a future where this is used by the circulation manager. We need people to stop using it now because of the performance hit, and once we have a replacement we need people to start using the replacement.

Reply:
:woman_facepalming: I didn't realize this was commented out.